### DISCUSSION TITLE
Bridge has no way to unbind events

### DISCUSSION THREAD

**Discussion by wiledal** [storyblok/monoblok#296] [Ideas]
`StoryblokBridgeV2` has `bridge.on(...)` but no `bridge.off(...)`.

This can be a major issue for obvious reasons.
If you're implementing an Event Emitter-like pattern, please allow us to also unsubscribe from events!

I also cannot find the repository for the bridge.js, is it closed source?

**Comment by fromthemills** [comment]
`registerStoryblokBridge` should also return a function to unregister

```js
const unregister = registerStoryblokBridge(...)
```

**Comment by maoberlehner** [comment]
Quick update: I've just worked out a potential solution for this and forwarded it to the responsible team internally.

**Reply by michasik** [reply]
is there any update on the matter?
